django/urls.py

```diff
@@ -14,3 +14,5 @@
     from django.contrib import admin
 
     urlpatterns += [path("admin/", admin.site.urls)]
+
+handler500 = 'core.api.views.custom_error'
```

Comment:
So every internal errors will be translated to bad request errors ? Can't we handle errors more specificaly ? In the issue, the error is a `ImproperMappingError`, why don't we catch this in `PreviewEndpoint` ?